### PR TITLE
docs: add overview page for simple DNN and fix some typos

### PR DESCRIPTION
**Pull request by serena-ruan**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 81 | 100 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0240603a-2697-46b0-ada2-4bbbb324270a) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 88 | 100 | 70 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b49fdc82-eb9f-4385-91f3-bf4deddebbcc) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/causal_inference/about.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcausal_inference%2Fabout.md) | 90 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/968ea2a4-db50-4773-bfc5-d1db5e928774) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/cognitive_services/CognitiveServices - Create Audiobooks.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcognitive_services%2FCognitiveServices%20-%20Create%20Audiobooks.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/56924351-2922-4067-90ce-f994691475f2) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/AzureSearchIndex - Met Artworks.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FAzureSearchIndex%20-%20Met%20Artworks.md) | 83 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ccff70e-463e-4fdc-ae19-a6ba2ca91d42) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews with Word2Vec.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews%20with%20Word2Vec.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00826bba-36fb-4dc7-9f75-1887b2197f5e) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews.md) | 91 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd65f5f1-cef0-43e5-93fa-acf6e5d054f7) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 81 | 100 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4cf8a6e2-c325-444f-b277-3ca9362fc5bb) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/656710cc4acd4cab7ac105216be2aa7bd5468c2d/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 88 | 100 | 70 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a8efef01-1194-4ed6-a956-db736cc71a9e) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: website/docs/features/simple_deep_learning/installation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+:::
+
+## 1. Reinstall horovod using our prepared script
+
+We build on top of torchvision, horovod and pytorch_lightning, so we need to reinstall horovod by building on specific versions of those packages.
+Download our [horovod installation script](https://mmlspark.blob.core.windows.net/publicwasb/horovod_installation.sh) and upload
+it to databricks dbfs.
+
+Add the path of this script to `Init Scripts` section when configuring the spark cluster.
+Restarting the cluster will automatically install horovod v0.25.0 with pytorch_lightning v1.5.0 and torchvision v0.12.0.
```
Comment (mhamilton723) [inline website/docs/features/simple_deep_learning/installation.md:17]:
```suggestion
Restarting the cluster automatically installs horovod v0.25.0 with pytorch_lightning v1.5.0 and torchvision v0.12.0.
```

File: website/docs/features/simple_deep_learning/installation.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+An alternative is installing the SynapseML jar package in library management section, by adding:
+```
+Coordinate: com.microsoft.azure:synapseml_2.12:0.11.0
+Repository: https://mmlspark.azureedge.net/maven
+```
+:::note
+If you install the jar package, you need to follow the first two cells of this [sample](./DeepLearning%20-%20Deep%20Vision%20Classification.md/#environment-setup----reinstall-horovod-based-on-new-version-of-pytorch)
+to make horovod recognize our module.
```
Comment (mhamilton723) [inline website/docs/features/simple_deep_learning/installation.md:34]:
```suggestion
If you install the jar package, follow the first two cells of this [sample](./DeepLearning%20-%20Deep%20Vision%20Classification.md/#environment-setup----reinstall-horovod-based-on-new-version-of-pytorch)
to ensure horovod recognizes SynapseML.
```

File: website/docs/features/simple_deep_learning/about.md [hunk 3]
```diff
@@ -1,42 +1,78 @@
...
+### Why Simple Deep Learning
+Creating a Spark-compatible deep learning system can be challenging for users who may not have a 
+thorough understanding of deep learning and distributed systems. Additionally, writing custom deep learning 
+scripts may be a cumbersome and time-consuming task.
+SynapseML aims to simplify this process by building on top of the [Horovod](https://github.com/horovod/horovod) Estimator, a general-purpose 
+distributed deep learning model that is compatible with SparkML, and [Pytorch-lightning](https://github.com/Lightning-AI/lightning),
+a lightweight wrapper around the popular PyTorch deep learning framework.
 
-## 1. Reinstall horovod using our prepared script
+SynapseML library offers a straightforward approach to leveraging deep learning in Spark-based distributed systems.
```
Comment (mhamilton723) [inline website/docs/features/simple_deep_learning/about.md:14]:
```suggestion
SynapseML's simple deep learning toolkit makes it easy to use modern deep learning methods in Apache Spark.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 82 | 100 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51015494-fd0d-4956-9c18-d3c8b28afef7) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/829d3d45-5958-4527-9b76-4384f0af0c32) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/causal_inference/about.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcausal_inference%2Fabout.md) | 90 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/393cbd30-84d6-4abc-aabb-78edad65b256) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/cognitive_services/CognitiveServices - Create Audiobooks.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcognitive_services%2FCognitiveServices%20-%20Create%20Audiobooks.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc2b5c0a-85e8-4d41-b50b-29e2d89ef7c3) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/AzureSearchIndex - Met Artworks.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FAzureSearchIndex%20-%20Met%20Artworks.md) | 83 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b72f32ac-8136-4c99-b310-e4de0d8b3970) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews with Word2Vec.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews%20with%20Word2Vec.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/17a788c4-fad2-4cec-a907-ba4075761cb4) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews.md) | 91 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a04232e8-5370-4d17-a2df-8cc265262ea0) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 81 | 100 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4d0cb53-5dcb-421d-9814-355c3a30f5d3) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/cde4ffc49cec20b3b4455463b511e2bab52de255/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 88 | 100 | 70 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/628a5732-bbf4-4570-b0da-905105e9cba2) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: website/docs/features/simple_deep_learning/about.md [hunk 4]
```diff
@@ -1,42 +1,77 @@
...
+Whether you are a data scientist, data engineer, or business analyst this project aims to make modern deep-learning methods easy to use for new domain-specific problems.
 
-We build on top of torchvision, horovod and pytorch_lightning, so we need to reinstall horovod by building on specific versions of those packages.
-Download our [horovod installation script](https://mmlspark.blob.core.windows.net/publicwasb/horovod_installation.sh) and upload
-it to databricks dbfs.
+### SynapseML's Simple DNN
+SynapseML goes beyond the limited support for deep networks in SparkML and provides out-of-the-box solutions for various common scenarios:
+- Visual Classification: Users can apply transfer learning for image classification tasks, using pre-trained models and fine-tuning them to solve custom classification problems.
+- Text Classification: SynapseML simplifies the process of implementing natural language processing tasks such as sentiment analysis, text classification, and language modeling by providing pre-built models and tools.
+- More to be added...
```
Comment (mhamilton723) [inline website/docs/features/simple_deep_learning/about.md:23]:
```suggestion
- And more coming soon
```

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
nit: Could you resolve some of the acrolinx issues for the about page? I took care of a few of them! (You might need to regen v11 docs afterwards so your fixed propagate there too)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 94 | 100 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f07fad93-8a24-42b8-9083-f05ecbf8e2f2) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d48d9fc-70ea-4b9b-8b9d-6fa8290b8297) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/documentation/estimators/causal/_causalInferenceDML.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Fdocumentation%2Festimators%2Fcausal%2F_causalInferenceDML.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b63f8dd-af6b-460a-a966-d79b8956214e) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/causal_inference/about.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcausal_inference%2Fabout.md) | 90 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0fbaa4d7-8aea-4970-8e31-ad70e16c069b) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/cognitive_services/CognitiveServices - Create Audiobooks.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcognitive_services%2FCognitiveServices%20-%20Create%20Audiobooks.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e006cbd-db27-4c4d-9848-9681626cc345) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/AzureSearchIndex - Met Artworks.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FAzureSearchIndex%20-%20Met%20Artworks.md) | 83 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/81323302-ee5c-4d9f-b754-d2c1763a34d7) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews with Word2Vec.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews%20with%20Word2Vec.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c756477d-bca7-4863-b402-142d9f9f7424) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews.md) | 91 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b8e4d21a-9d0a-4c50-8f4c-efa252ca132f) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 94 | 100 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b378081-149a-409a-bcb2-37316f198de4) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/6d1557e51c44bc8d27415fa6b1ba4085bd4cbfb8/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6618ca6a-9e60-4d17-9387-47c7d77b2ff6) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 97 | 100 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ec4db83-83ff-4686-b47e-cfb8317b8baa) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fdocs%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b9376ee5-7b79-40d1-aad1-0dbdd73cd1f5) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/documentation/estimators/causal/_causalInferenceDML.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Fdocumentation%2Festimators%2Fcausal%2F_causalInferenceDML.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ad800246-d523-4a63-86c7-966f667f1975) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/causal_inference/about.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcausal_inference%2Fabout.md) | 90 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe108bde-bdba-46ef-a3f2-51d5cf7d7ade) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/cognitive_services/CognitiveServices - Create Audiobooks.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fcognitive_services%2FCognitiveServices%20-%20Create%20Audiobooks.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7b8b8abd-2bfe-4b1e-9ce6-92284e94efbd) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/AzureSearchIndex - Met Artworks.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FAzureSearchIndex%20-%20Met%20Artworks.md) | 83 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f0d27c2-43b2-423a-95a9-3c841f40f8b0) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews with Word2Vec.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews%20with%20Word2Vec.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22476a4c-a540-4979-bd23-a212d339a466) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/other/TextAnalytics - Amazon Book Reviews.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fother%2FTextAnalytics%20-%20Amazon%20Book%20Reviews.md) | 91 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/442130a0-9e68-42dd-8622-aebe4b53451d) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/about.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Fabout.md) | 97 | 100 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a349843a-35a9-4824-a38f-610e910878aa) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.11.0/features/simple_deep_learning/installation.md](https://github.com/microsoft/SynapseML/blob/f6b7fc506735f82ea4f5837da404b49c999c6f80/website%2Fversioned_docs%2Fversion-0.11.0%2Ffeatures%2Fsimple_deep_learning%2Finstallation.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/400bb189-8c03-4425-9d42-ac41d1b0cd58) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**